src/generateTrackerData.ts

```diff
@@ -0,0 +1,60 @@
+import { IAnalyticsTrackerOptions } from './types';
+
+export const generateTrackerData = (
+  options: IAnalyticsTrackerOptions,
+): { metricName: string; duration?: number; value?: any } => {
```

Comment:
`duration` in v5 was deprecated, now only `data` is supported.

Reply:
I think you wanna use `IAnalyticsTrackerOptions` instead, and then just update `IAnalyticsTrackerOptions` by removing `duration` from there.